### PR TITLE
networks: add Nov 15 checkpoint

### PR DESCRIPTION
**Pull request by pinheadmz**
See https://github.com/Bitcoin-ABC/bitcoin-abc/commit/651ac4461c2c92952df39f75a9d177c746e60b57

Sets checkpoint at block height 556767 to prevent rollback of recent hard fork. Tested with local SPV node.

### CODE REVIEW

File: lib/protocol/networks.js [hunk 1]
```diff
@@ -120,7 +120,9 @@ main.checkpointMap = {
   510000: b('040e6b1f2f4cb198a5780d366bf81e591de257642b9267030000000000000000'),
   525000: b('c994fba2bf168333fd969bcfa64f03ca1b62074f9a8f1b010000000000000000'),
   // Monolith Activation
-  530359: b('0391c40195cf8ae3436f3955f1a8444f07468fd08bda1a010000000000000000')
+  530359: b('0391c40195cf8ae3436f3955f1a8444f07468fd08bda1a010000000000000000'),
+  // Nov 15 2018 Upgrade fork:
```
Comment (tuxcanfly) [inline lib/protocol/networks.js]: Nit: update to 'Magnetic Activation'

Comment (pinheadmz) [reply lib/protocol/networks.js]: er, Anomaly? so `// Nov 15 2018  Magnetic Anomaly Activation` ?

Comment (tuxcanfly) [reply lib/protocol/networks.js]: OK, or just `Magnetic Anomaly Activation`.

**Comment by pinheadmz** [comment]
updated & squashed

**Pull request merged**